elastic/kibana
# [Uptime] Localization

## Summary

Resolves #27524.

Add translations to uptime plugin.

## Testing this PR
- Load the Uptime plugin from the Kibana sidebar without any heartbeat data
- Ensure the empty state loads correctly and the copy looks right
- Start logging some heartbeat data for up/down services
  - Check the Overview page, as well as a Monitor page
